networking/kvm.go

```diff
@@ -130,8 +136,12 @@ func kvmSetupNetAddressing(network *Networking, n activeNet, ifName string) erro
 	if err := ip.EnableIP4Forward(); err != nil {
 		return errwrap.Wrap(errors.New("failed to enable forwarding"), err)
 	}
+
+	// patch plugin type only for single IPAM run time, then revert this change
```

Comment:
With this method I can run underlying IPAM plugin to obtain IP configuration.
Without reverting this value after process, we loosed information about original plugin (saved later in .json) and this was a reason why `gc` without that does not work correctly in this configuration.